### DISCUSSION TITLE
Running in Local Environment

### DISCUSSION THREAD

**Discussion by dids-reyes** [skedaddl3/letters_to_casper#117] [Set Up]
**Frontend Environment Variables**

```env
REACT_APP_API_URL=''
REACT_APP_API_KEY=''
REACT_APP_BASE_URL=''
REACT_APP_CLNDR_API_KEY=''
```
Add `.env` in the root of the project directory containing the values above

For Example:
```
letters_to_casper/
├── build
├── docs
├── node_modules
├── public
├── src
├── tests
├── README.md
├── .gitignore
├── package-lock.json
├── package.json
└── .env
```

Run React App: 

```shell
npm run start
```

**Backend Environment Variables**

Backend Repository: [ltc-service](https://github.com/dids-reyes/ltc-service)
```env
url=''
API_KEY=
IP_INFO_TOKEN=''
email_issuer='l'
email_receiver=''
app_password=''
sentry_dsn=''
```

Run Server Command: 

```shell
npm start
```

<hr/>

**To view incoming Letters / Pending / For Approval**

Backend is sending data to channel #letters in Discord
Join Discord Channel: [Letters to Casper](https://discord.gg/pQf2mAHe4u)

**Comment by christiandls444** [comment]
up
